### PR TITLE
3.0/feature/replace age range

### PR DESCRIPTION
**Pull request by annalinneajohansson**

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-1xrct3a Vissa filter för sök i Aktivitetskalendern funkar ej](https://app.clickup.com/t/1xrct3a)

**Comment by helsingborgsstad** [comment]
Task linked: [CU-3dux7kk Hantera nya query vars i PHP](https://app.clickup.com/t/3dux7kk)

**Comment by helsingborgsstad** [comment]
Task linked: [CU-3dux7my Skicka nya query vars från JS till PHP](https://app.clickup.com/t/3dux7my)

**Comment by helsingborgsstad** [comment]
Task linked: [CU-3dux7xx Rensa upp gammal kod som inte körs i event modulen](https://app.clickup.com/t/3dux7xx)

File: source/php/Helper/QueryEvents.php [hunk 1]
```diff
@@ -71,17 +73,48 @@ public static function getEventsByInterval($params, $page = 1)
...
         WHERE $wpdb->posts.post_type = %s
         AND $wpdb->posts.post_status = %s
         AND ($db_table.start_date BETWEEN %s AND %s OR $db_table.end_date BETWEEN %s AND %s) ";
 
+		if ( $ageMin ) {
+			$query .= "AND ( 
+				( 
+				  ( mt2.meta_key = 'age_group_from' AND mt2.meta_value <= $ageMin )
+				) 
+				OR
```
Comment (annalinneajohansson) [inline source/php/Helper/QueryEvents.php:96]: We need a third condition to match events where mt2.meta_key IS NULL but has age_group_to set (regardless of value, simply not IS NULL). Same thing in reverse for the $ageMax check on :104

**Pull request merged**